src/core/Akka.Remote/EndpointManager.cs

```diff
@@ -324,12 +330,39 @@ private ICancelable PruneTimerCancelleable
                 {
                     return _pruneTimeCancelable = Context.System.Scheduler.ScheduleTellRepeatedlyCancelable(PruneInterval, PruneInterval, Self, new Prune(), Self);
                 }
-                return null;
+                return _pruneTimeCancelable;
+            }
+        }
+
+        private Dictionary<IActorRef, AkkaProtocolHandle> _pendingReadHandoffs = new Dictionary<IActorRef, AkkaProtocolHandle>();
+        private Dictionary<IActorRef, List<InboundAssociation>> _stashedInbound = new Dictionary<IActorRef, List<InboundAssociation>>();
+
+
+        private void HandleStashedInbound(IActorRef endpoint, bool writerIsIdle)
```

Comment:
Modified this method to support the new `writerIsIdle` field, which refers to whether or not the `ReliableEndpointWriter` is in an `Idle` state or not and affects how we flush messages onto the network.

Originally this method appeared further below inside the `EndpointManager` class, but I moved it up here so I could read this code alongside the JVMs.